src/Logic.hs

```diff
@@ -219,37 +218,21 @@ runAction config =
         Right integratedSha -> pure $ Right integratedSha
     TryPromote prBranch sha -> do
       ensureCloned config
-      forcePushResult <- Git.forcePush sha prBranch
-      case forcePushResult of
-        PushRejected _ -> pure forcePushResult
-        PushOk -> do
-          -- TODO: Find a safer way to make sure Github doesn't get confused
-          -- by 2 pushes close together, the delay is all arbitrary and not nice.
-          -- See https://github.com/channable/hoff/issues/196
-          Time.sleepMicros pushDelayMicroseconds
-          Git.push sha (Git.Branch $ Config.branch config)
+      Git.pushAtomic [AsRefSpec (ForceRefSpec (sha, prBranch)), AsRefSpec (sha, Git.Branch $ Config.branch config)]
```

Comment:
Nit: the ref specs are a bit noisy, can you insert line breaks here so that we have one line per thing pushed? Makes it a bit easier to visually digest